petab/v2/petab1to2.py

```diff
@@ -209,6 +223,9 @@ def create_experiment_id(sim_cond_id: str, preeq_cond_id: str) -> str:
                 measurement_df, get_dest_path(measurement_file)
             )
 
+    new_yaml_file = output_dir / Path(yaml_file).name
+    write_yaml(new_yaml_config, new_yaml_file)
```

Comment:
Might simplify a little to move `output_dir.mkdir(...)` into `write_yaml` but fine as is

Reply:
Will do it in a separate PR.